packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -19,7 +19,7 @@ interface Props extends CommonInteractionProps {
   onSelectionStart?: () => void;
   onSelectionChange?: (points: Selection) => void;
   onSelectionEnd?: (points: Selection) => void;
-  children: (points: Selection) => ReactElement;
+  children: (points: Selection) => ReactNode;
```

Comment:
This, along with the extra fragment around `children()` below (for TS), allows the render prop to return `null`, `false`, etc.